src/GitHub.App/ViewModels/Dialog/Clone/RepositoryCloneViewModel.cs

```diff
@@ -133,7 +140,11 @@ public async Task InitializeAsync(IConnection connection)
                 EnterpriseTab.Initialize(enterpriseConnection);
             }
 
-            if (connection == enterpriseConnection)
+            if (connection == gitHubConnection)
+            {
+                SelectedTabIndex = 0;
```

Comment:
Curious, are you adding this for the sake of the reading programmer? or is this function called multiple times now?

Reply:
After conversation, the function is not re-entrant, it's just left there to be explicit.